src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -1136,6 +1139,98 @@ export const GlobalProvider = memo(
 
         const [connectingFrom, setConnectingFrom] = useState<OnConnectStartParams | null>(null);
 
+        const getNodesBoundingBox = useCallback(() => {
```

Comment:
This function will return `{x:Infinity, y:Infinity, width:-Infinity, height:-Infinity}` for any empty chain.

Could you please add a check that returns `undefined` if there are no nodes? `exportViewportScreenshot` could then check for `undefined` and return early.

On that note: Maybe we should *also* disable the export button if the chain is empty?

Reply:
Good catch, completely forgot to test that.

Have added the check and disabled the button when there are no nodes.